cms/static/require-config.js

```diff
@@ -77,7 +77,7 @@ require.config({
             // so that require doesn't fall over
             "js/src/tender_fallback"
         ],
-        "mathjax": "//cdn.mathjax.org/mathjax/2.4-latest/MathJax.js?config=TeX-MML-AM_HTMLorMML-full&delayStartupUntil=configured",
+        "mathjax": "//cdn.mathjax.org/mathjax/2.6-latest/MathJax.js?config=TeX-MML-AM_HTMLorMML-full&delayStartupUntil=configured",
```

Comment:
Because it is not useful (and could be harmful due to `collectstatic`) to put the mathjax inside the platform.